miden-tx/src/tests.rs

```diff
@@ -65,10 +65,14 @@ fn transaction_executor_witness() {
         vm_processor::execute(tx_witness.program(), stack_inputs, &mut host, Default::default())
             .unwrap();
 
-    let (advice_provider, _event_handler) = host.into_parts();
+    let (advice_provider, _event_handler, output_notes) = host.into_parts();
```

Comment:
nit: I think the returned tuple should be `(advice_provider, _account_delta, output_notes)`.

Reply:
removed `event_handler`